test/e2e_env/kubernetes/bootstrap/corefile_template.go

```diff
@@ -0,0 +1,84 @@
+package bootstrap
```

Comment:
The test is nice, but I don't think this should go into `e2e_env/kubernetes`.

The idea of `test/e2e_env/kubernetes` is to be able to parallelize tests. This test breaks this promise. Yes, we use Kuma2 here, but imagine we also have another test that needs to deploy Kuma with a custom config using Kuma2. It cannot be then used in parallel with this test.

Another disadvantage is that we need to run a second Kube cluster just for this test. When developing a kube e2e test, usually you focus the suite you are writing and run `make test/e2e-kubernetes`. Forcing running a second kube cluster every single time prolongs the development loop.

I'd move this to `test/e2e`.

Reply:
Yeah, that'll make more sense, you may go ahead moving it and revert the changes in `e2e.mk`. Thanks a lot.
I wasn't sure if it's good to place it under `e2e_env/kubernetes`, but I felt like it's also not very suitable to but in `test/e2e ` when compared to existing tests.

Reply:
A new PR has been created to conduct this change, please review:
https://github.com/kumahq/kuma/pull/8736